lychee.example.toml

```diff
@@ -1,59 +1,97 @@
-###
-### Display
-###
+#############################  Display  #############################
+
 # Verbose program output
 verbose = false
 
-# Show progress
-progress = false
+# TODO
+no_progress = false
+
+# TODO
+output = "report.md"
+
+#############################  Cache  ###############################
+
+# TODO
+cache = true
 
+# TODO
+# Error: invalid type: string "2d", expected struct Duration for key `max_cache_age`
+# max_cache_age = "2d"
+
+#############################  Runtime  #############################
 
-###
-### Runtime
-###
 # Number of threads to utilize.
 # Defaults to number of cores available to the system if omitted.
-#threads = 2
+threads = 2
 
 # Maximum number of allowed redirects
 max_redirects = 10
 
+# TODO
+max_retries = 2
+
+# TODO
+max_concurrency = 14
+
+#############################  Requests  ############################
 
-###
-### Requests
-###
 # User agent to send with each request
 user_agent = "curl/7.71.1"
 
 # Website timeout from connect to response finished
 timeout = 20
 
-# Comma-separated list of accepted status codes for valid links.
-# Omit to accept all response types.
-#accept = "text/html"
+# TODO
+retry_wait_time = 2
+
+# TODO
+accept = "403, 403"
 
 # Proceed for server connections considered insecure (invalid TLS)
 insecure = false
 
 # Only test links with the given schemes (e.g. https)
 # Omit to check links with any scheme
-#scheme = [ "https" ]
+scheme = [ "https" ]
+
+# TODO
+require_https = false
 
 # Request method
 method = "get"
 
 # Custom request headers
 headers = []
 
+# TODO # Remap URI matching pattern to different URI
+remap = [ "https://example.com http://example.invalid" ]
+
+# TODO
+# Error: invalid type: unit variant, expected newtype variant for key `base`
+# base = "https://example.com"
+
+# TODO
```

Comment:
```
HTTP basic auth support. This will be the username and password passed to the authorization HTTP header. See https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Authorization
```